src/views/ont/OntReception.vue

```diff
@@ -1,7 +1,9 @@
 <template>
   <div class="reception">
     <alert ref='alert'></alert>
-
+    <b-modal v-model="importing" :hide-footer=true :hide-header=true :no-close-on-backdrop=true>
```

Comment:
Think these `:hide-footer=true` can just be `hide-footer`,  the : is for dynamic biinding